addons/search-profile/addon.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Search profile",
+  "name": "Search for users",
```

Comment:
```suggestion
  "name": "User search",
```

Reply:
Sounds more like you're searching projects a user made. That's why I changed it to "search _for_ users". Oh wait, that's more of a phrase than a noun, though. Hmm...

Reply:
I like Search for users

Reply:
Maybe "username search"?